README.md

```diff
@@ -85,7 +88,22 @@ buildscript {
 
 apply plugin: "org.jlleitschuh.gradle.ktlint"
 ```
+</details>
+<details open>
+<summary>Kotlin</summary>
 
+```groovy
+buildscript {
+  repositories {
+    maven("https://plugins.gradle.org/m2/")
+  }
+}
+
+plugins {
+  id "org.jlleitschuh.gradle.ktlint" version "<current_version>"
+}
+```
+</details>
```

Comment:
This block should use kotlin lang :slightly_smiling_face: :
```
```kotlin
```

Another thing that this block should show how-to use this plugin via old classpath approach using kotlin script:
```kotlin
buildscript {
  repositories {
    maven("https://plugins.gradle.org/m2/")
  }
  dependencies {
    classpath("org.jlleitschuh.gradle:ktlint-gradle:<current_version>")
  }
}

apply(plugin = "org.jlleitschuh.gradle.ktlint")
```

Reply:
Thanks for reviewing @Tapchicoma, I will correct.

Reply:
@Tapchicoma can you please tell me am I supposed to entirely replace my block with the one you sent?

**from**
```
buildscript {
  repositories {
    maven("https://plugins.gradle.org/m2/")
  }
}

plugins {
  id("org.jlleitschuh.gradle.ktlint") version "<current_version>"
}
```

to
```
buildscript {
  repositories {
    maven("https://plugins.gradle.org/m2/")
  }
  dependencies {
    classpath("org.jlleitschuh.gradle:ktlint-gradle:<current_version>")
  }
}

apply(plugin = "org.jlleitschuh.gradle.ktlint")
```

Reply:
@nuhkoca yes, please fully replace your block with what I've proposed